common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
@robrap @cahrens It turned out that the content of the `msg` will be html or an empty string for all these templates that we've updated. Since we already put a condition for `msg` i.e `if msg` so i think we're good with HTML only. Sorry for false alarm. Thanks for explanation and making them clear.